packages/trader/src/Modules/CFD/Containers/props.types.ts

```diff
@@ -36,4 +36,65 @@ export type TPasswordResetAndTradingPasswordManager = {
     email: string;
     platform: 'mt5' | 'dxtrade';
     account_group: 'real' | 'demo';
+    toggleModal?: () => void;
+};
+
+export type TCountdownComponent = {
+    count_from: number;
+    onTimeout: (value?: boolean) => void;
+};
+
+export type TCFDPasswordReset = {
+    sendVerifyEmail: () => Promise<any>;
+    account_type: string;
+    account_group: 'real' | 'demo';
+    server: string;
+    password_type: string;
+};
+
+export type TCFDPasswordSuccessMessage = {
+    toggleModal: () => boolean;
```

Comment:
As I chased this method, this gets `togglePasswordManagerModal` method from the CFDDashboard component which does not have any method, so I think it should be `() => void `